### PR TITLE
910 guideline for roles

### PR DESCRIPTION
**Pull request by AnaCascante**
## Title
#910 

## Describe your changes: 
New document to clarify the roles of the team members

## Type of changes:
Describe the responsability of each role

## Checklist before requesting a review

* [x] I created a new branch before starting with this ticket?
* [x] I commented the issue after finish in the Frontend Team Board?
* [x] I moved the card to Quality Assurance after finish?

## Did you run into any issues?

- No, issues encountered.

## Screenshots:

- No screenshots to include.

## Feedback

- Yes, I want feedback.

### CODE REVIEW

**Review by Spookyrumble — APPROVED** [review]
LGTM

File: documentation/Team-roles-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+4. **Continuous Improvement:**
+   - Encourage the team to regularly reflect on their processes and identify areas for improvement.
+   - Help the team adapt to change and continuously refine their working practices.
+
+### Quality Assurance (QA):
+
+1. **Responsability:**
+   - Quality Assurance serves as the primary reviewer for the project, ensuring thorough assessment and validation of deliverables.
+   - Review the code changes and provide feedback.QA may verify that the reported issue has been adequately addressed and resolved.
```
Comment (kyrregjerstad) [inline documentation/Team-roles-guidelines.md:27]:
```suggestion
   - Review the code changes and provide feedback. QA may verify that the reported issue has been adequately addressed and resolved. 
```

File: documentation/Team-roles-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+4. **Continuous Improvement:**
+   - Encourage the team to regularly reflect on their processes and identify areas for improvement.
+   - Help the team adapt to change and continuously refine their working practices.
+
+### Quality Assurance (QA):
+
+1. **Responsability:**
+   - Quality Assurance serves as the primary reviewer for the project, ensuring thorough assessment and validation of deliverables.
+   - Review the code changes and provide feedback.QA may verify that the reported issue has been adequately addressed and resolved. 
+   - QA is responsible for merging, and if the issue is properly linked, it should automatically close itself. However, it sometimes encounters bugs.
```
Comment (kyrregjerstad) [inline documentation/Team-roles-guidelines.md:28]:
```suggestion
QA handles the merging process, and if the issue is correctly linked, it should close automatically. However, this process can occasionally experience bugs.
```

File: documentation/Team-roles-guidelines.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+   - Ensure that quality is maintained throughout the development process.
+   - Collaborate with developers to understand user stories and acceptance criteria.
+   - Establish a dedicated channel for Quality Assurances to collaborate and communicate effectively.
```
Comment (kyrregjerstad) [inline documentation/Team-roles-guidelines.md:31]:
```suggestion
   - Establish a dedicated channel for all Quality Assurance teams to collaborate and communicate effectively.
```

File: documentation/Team-roles-guidelines.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+   - Provide feedback to the development team on the quality of deliverables.
+
+5. **Communication:**
+   - Facilitating transparent and open communication with both Scrum Masters and developers to nurture a collaborative working environment.
+   - Establish a dedicated channel for Quality Assurances to collaborate and communicate effectively.
+
+### Developer:
+
+1. **Coding and Implementation:**
+   - Write clean, maintainable, and efficient code based on requirements.
```
Comment (kyrregjerstad) [inline documentation/Team-roles-guidelines.md:53]:
```suggestion
   - Write clean, maintainable, and efficient code based on the requirements.
```

File: documentation/Team-roles-guidelines.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+   - Foster a collaborative environment, sharing knowledge and skills with team members.
+
+3. **Adaptability:**
+   - Embrace changes in requirements and be flexible in adapting to evolving project needs.
+   - Continuously learn and apply new technologies and best practices.
+
+4. **Code Review and Refactoring:**
+   - Participate in code reviews to ensure code quality and adherence to coding standards.
+   - Proactively identify opportunities for code refactoring to improve maintainability.
+   - Developers are required to verify that their issues have been merged and closed. Do NOT delete issues.
```
Comment (kyrregjerstad) [inline documentation/Team-roles-guidelines.md:67]:
```suggestion
   - Developers are required to verify that their PRs have been merged and issues closed. Do NOT delete issues. A branch can safely be deleted when the PR is merged.
```

**Comment by kyrregjerstad** [comment]
Nice work, I added some comments :)

File: documentation/Team-roles-guidelines.md (hunk 1 above)
Comment (AnaCascante) [reply documentation/Team-roles-guidelines.md]: made the suggested change.

File: documentation/Team-roles-guidelines.md (hunk 3 above)
Comment (AnaCascante) [reply documentation/Team-roles-guidelines.md:31]: maybe better: channel for QAs from all teams

**Review by kyrregjerstad — APPROVED** [review]

**Pull request merged**